{"name": "执行用例", "status": "broken", "statusDetails": {"message": "AttributeError: 'Test_pet_case' object has no attribute 'headers'", "trace": "self = <test_case.test_dome.Test_pet_case object at 0x7f7cb2336e50>\npath = '/updataUserInfo', reqData = {'tel': 11111111111}\n\n    @allure.story('执行用例')\n    @allure.title('执行用例')\n    @pytest.mark.run(order=1)\n    @pytest.mark.parametrize('path',[read_yaml(os.getcwd(),'apiData', 'testData.yaml')['updataUserInfo']['path']])\n    @pytest.mark.parametrize('reqData',[read_yaml(os.getcwd(),'apiData','testData.yaml')['updataUserInfo']['reqData']])\n    def test_creatPet(self,path,reqData):\n        url = self.apiurl + path\n>       resMsg = requ(self.headers).post(url=url,jsonParams=reqData)\nE       AttributeError: 'Test_pet_case' object has no attribute 'headers'\n\ntest_case/test_dome.py:41: AttributeError"}, "parameters": [{"name": "reqData", "value": "{'tel': 11111111111}"}, {"name": "path", "value": "'/updataUserInfo'"}], "start": 1680172670573, "stop": 1680172686213, "uuid": "611e3fce-f0ac-47be-a085-640b40388c43", "historyId": "167f808980cb0e9d31e432ba244eb02b", "testCaseId": "2e175a4edcd761e9d5708c0b071ccfb3", "fullName": "test_case.test_dome.Test_pet_case#test_creatPet", "labels": [{"name": "story", "value": "执行用例"}, {"name": "tag", "value": "run(order=1)"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_dome"}, {"name": "subSuite", "value": "Test_pet_case"}, {"name": "host", "value": "localhost"}, {"name": "thread", "value": "38908-Thread-7"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_dome"}]}